### PR TITLE
Amend JENKINS-24110 fix to more politely specify what was not being run

### PR DESCRIPTION
**Pull request by jglick**
Amends #1352.

### Proposed changelog entries

* None needed, just change to logging.

@reviewbybees

### CODE REVIEW

File: core/src/main/java/hudson/model/Executor.java [hunk 1]
```diff
@@ -355,6 +355,10 @@ public SubTask call() throws Exception {
                         LOGGER.log(FINE, getName()+" grabbed "+workUnit+" from queue");
                     SubTask task = workUnit.work;
                     Executable executable = task.createExecutable();
+                    if (executable == null) {
+                        String displayName = task instanceof Queue.Task ? ((Queue.Task) task).getFullDisplayName() : task.getDisplayName();
+                        LOGGER.log(WARNING, "{0} cannot be run (for example because it is disabled)", displayName);
+                    }
```
Comment (oleg-nenashev) [inline core/src/main/java/hudson/model/Executor.java]: Probably you should return here then

Comment (jglick) [inline core/src/main/java/hudson/model/Executor.java]: I thought that was exactly what we did not want to do, based on your own earlier comment:

> In such case Jenkins starts the workUnit in order to report results to console outputs.

Comment (oleg-nenashev) [inline core/src/main/java/hudson/model/Executor.java]: agreed

**Review by oleg-nenashev — APPROVED** [review]
🐝

**Comment by jglick** [comment]
@reviewbybees done

**Comment by oleg-nenashev** [comment]
Will merge tomorrow if nobody votes against

**Pull request merged**